content/en/synthetics/browser_tests/test_results.md

```diff
@@ -10,35 +10,35 @@ further_reading:
   text: "Monitor Core Web Vitals with Synthetic Monitoring"
 ---
 
+A test result is the result of a Datadog Synthetic test execution. For browser tests, a test result corresponds to the execution of the test at some point in time, on one specific location, on a specific browser and type of device.
 
-Test results and performance data are accessed from the **Step Results** section when clicking on a specific test result. You can also use **Sample Results** as a way to quickly look into the latest failed test runs and compare them with recent successful ones.
+You can use the **Sample Results** section as a way to quickly look into the latest failed test runs and compare them with recent successful ones. You can also scroll down to the **Test Results** section of the page, click on any test result to look into it more thoroughly.
 
-## Test failure
+[Browser test results](#test-results) consist of a variety of components such as [screenshots](#screenshots), [page performance date](#page-performance), [errors](#errors), [resources](#resources), [backend traces](#backend traces), and more that can help you troubleshoot your [test failure](#test-result-failure).
 
-A test is considered `FAILED` if it does not satisfy its assertions or if a step failed for another reason. You can troubleshoot failed runs by looking at their screenshots, checking for potential [errors](#errors) at the step level, and looking into [backend traces](#traces) generated by their steps.
+## Test results
 
-Common browser test errors include:
+The following test result characteristics are displayed at the top of every browser test result:
 
-`Element located but it's invisible` 
-: The element is on the page but cannot be clicked on—for instance, if another element is overlaid on top of it.
+- Status
+- Starting URL
```

Comment:
@MargotLepizzera when I look at Sample Results of a browser test, I see Starting URL but in Test Results, I don't see Starting URL. Should we remove this entry?

Reply:
We can leave it - we might actually add the starting URL to the test results list at some point 😄